### ISSUE TITLE
Only show symbol when edittext has focus

### ISSUE THREAD

**Issue by evolutionise** [scottyab/showhidepasswordedittext#19]
Hi,

I'd really like to have the option to only show the icon when the edit text has focus. If this is something you'd be interested in adding, I'm happy to submit a pull request.

**Comment by scottyab** [comment]
Hey @evolutionise, yeah that would be useful, thanks. Maybe make it optional with method call and a custom xml attribute . i.e `showPasswordIconWhenFocus` (or a more concise name if you can think of one)

**Comment by evolutionise** [comment]
Hey @scottyab - I'm finding that even on the master branch, the show-hide functionality isn't working in most of the sample edit texts, i.e. there is text in the edit text, it is shown in the dotted form, and tapping the eye symbol doesn't change it to clear text. It only works in the TextInputLayout.  

I had a problem with the gradle version having expired (from using a beta).  I've tried with 2.0.0 (the closest version to the beta used) and 2.1.2 (current version), but neither of them fix the problem.  

Do you have any thoughts about why this is? I can send you a screen capture of the issue if that would help.

**Comment by scottyab** [comment]
Hmm ok,  sounds strange. A video would help more than screen grabs. Thanks

**Comment by scottyab** [comment]
I'm seeing this too :(

**Comment by evolutionise** [comment]
Ahh I see you're encountering the same problem.  Here's the link to a video of it anyway: https://drive.google.com/open?id=0B4dWTi2C0bQLY0d5VW1qZHJnYzg

**Comment by evolutionise** [comment]
I can still submit a PR for the work to show focus if you like - as I'm using the TextInputLayout version which still works, I can verify that it's working correctly in there.

**Comment by scottyab** [comment]
Yes. Thanks I'll take a look at the other focus issue separately in couple
days when I have time. (that is if someone does figure it out beforehand)

**Comment by evolutionise** [comment]
It seems that the issue is that the touch target is tiny and in the top right corner - perhaps the same issue as https://github.com/scottyab/showhidepasswordedittext/issues/16.

**Comment by scottyab** [comment]
@evolutionise I've fixed the issue of the tiny touch target and added a new attribute `app:additionalTouchTargetSize` if you want to make it bigger
